src/Nethermind/Ethereum.Test.Base/AccountState.cs

```diff
@@ -21,16 +21,23 @@ public class AccountState
         [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
         public Dictionary<UInt256, byte[]> Storage { get; set; }
 
-        public static AccountState GetFromAccount(Address address, WorldState stateProvider)
+        public static AccountState GetFromAccount(Address address, WorldState stateProvider, Dictionary<Address, Dictionary<UInt256, byte[]>> storages)
         {
             var account = stateProvider.GetAccount(address);
             var code = stateProvider.GetCode(address);
-            return new AccountState()
+            var accountState = new AccountState()
             {
                 Nonce = account.Nonce,
                 Balance = account.Balance,
                 Code = code.Length == 0 ? null : code
             };
+
+            if (storages.TryGetValue(address, out var storage))
```

Comment:
what is the goal of this change?

Reply:
Alloc output needs storage values, so I created T8NToolTracer in order to collect storages while executing contracts